services/application.js

```diff
@@ -346,7 +347,58 @@ class Application {
         })
     }
 
+    async sendEmailAfterApproveApplication(context, application) {
+        // org owner email
+        let org = await this.organizationService.getOrganizationByID(application?.organization?._id);
+        let org_owner_email = null
+        let org_owner_id = org?.owner
+        if(org_owner_id){
+            let org_owner = await this.userService.getUserByID(org_owner_id);
+            if(org_owner?.email){
+                org_owner_email = org_owner?.email
+            }
+        }
+
+        // concierge email
+        let concierge_email = null
+        let org_concierge_id = org?.concierges
+        if(org_concierge_id){
+            let org_concierge = await this.userService.getUserByID(org_concierge_id);
+            if(org_concierge?.email){
+                concierge_email = org_concierge?.email
+            }
+
+        }
+        
+        // admin email
+        let admin_user = await this.userService.getAdmin();
+        let admin_email = ""
+
+        for(let i of admin_user){
+            admin_email = admin_email + " ; " + i.email
+        }
+
+        // cc emil
+        let cc_email
+        if(!concierge_email){
+            cc_email = concierge_email
+        }else{
+            cc_email = admin_email
+        }
+
 
+        await this.notificationService.approveQuestionNotification(application?.applicant?.applicantEmail,
+            // Organization Owner and concierge assigned/Super Admin
+            `${org_owner_email} ; ${cc_email}`,
+        {
+            firstName: application?.applicant?.applicantName
+        }, {
+            study: application?.studyAbbreviation,
+            doc_url: config.submission_doc_url,
```

Comment:
please add submission_doc_url to config.js, the env.template, and the readme doc